tests/cypress/e2e/unit_tests/rbac_fleet.spec.ts

```diff
@@ -16,87 +16,66 @@ import 'cypress/support/commands';
 import * as cypressLib from '@rancher-ecp-qa/cypress-library';
 import { qase } from 'cypress-qase-reporter/dist/mocha';
 
+export const baseUser      = "base-user"
```

Comment:
Again, this enforces going up and down, but up to you.

Reply:
Okay, I will remove it.